test/simplifyExpression/basicsSearch/basicsSearch.test.js

```diff
@@ -43,5 +31,5 @@ describe('simplify basics', function () {
     // exponent to 0 -> 1
     ['(x+3)^0', '1'],
   ];
-  tests.forEach(t => testSimplify(t[0], t[1]));
+  tests.forEach(t => testSimplify(t[0], t[1], basicsSearch));
```

Comment:
I feel we probably don't need to keep these tests here if they're duplicated elsewhere.

I think the two best options are:

- do what you were saying and "test expressions that require multiple simplifications in this file"
- just delete this file, since it's redundant (we have similar testing for a few other places in the codebase already)